packages/eslint-config-eventbrite-legacy/package.json

```diff
@@ -5,10 +5,10 @@
   "main": "index.js",
   "scripts": {
     "git-tag-version": "node ./_scripts/git-tag-version.js",
-    "lint": "eslint .",
-    "test": "npm run lint",
-    "preversion": "npm test",
-    "version": "npm run git-tag-version",
+    "lint": "eslint rules",
+    "test": "yarn run lint",
```

Comment:
nit you can drop the run from yarn commands

Reply:
Yeah true. So I do this thing where when I'm in the command line I do `yarn [scriptName]` but in scripts I do the full `yarn run [scriptName]`. This is just in case we happen to pick a script name that unknowingly collides w/ an actual yarn script. Maybe they add something in the future. It's just a bit of over-worried future-proofing...